src/features/surveys/components/SurveyUnlinkedCard.tsx

```diff
@@ -36,7 +42,16 @@ const SurveyUnlinkedCard = ({ orgId, surveyId }: SurveyUnlinkedCardProps) => {
                   }
                   subheader={messages.unlinkedCard.description()}
                 >
-                  <></>
+                  <NextLink
+                    href={`/organize/${orgId}/campaigns/${campId}/surveys/${surveyId}/submissions?filter=linked`}
+                    passHref
+                  >
+                    <Link>
+                      {messages.unlinkedCard.openLink({
+                        numUnlink: unlinkedSubmitters,
+                      })}
```

Comment:
No need to change this, but this is the type of situation where I would use `<Msg/>` instead of `useMessages()`.